docs/sphinxext/numpy_ext/docscrape.py

```diff
@@ -6,7 +6,10 @@
 import textwrap
```

Comment:
I did remove `sphinxext/numpydoc`, in favor of installing the one from pip. Didn't touch / test what `numpy_ext` is doing.